src/components/RadioButtonGroup/RadioButtonGroup.story.js

```diff
@@ -47,7 +47,14 @@ const RadioButtonGroupWithState = ({ value: initial, children, ...props }) => {
     e.persist();
     setValue(e.target.value);
```

Comment:
I understand you didn't change this in this PR, but I'm still curious why we need to use `persist` why doesn't the event get propagated without it?

Reply:
I just tested without and it continued to work, so it doesn't seem to be necessary. I've removed it.